### PR TITLE
chore: version

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+var (
+	// The following fields are populated at build time using -ldflags -X.
+	_buildVersion = "unknown"
+	_buildGitRevision = "unknown"
+
+	_buildGoVersion = runtime.Version()
+)
+
+// Version produces a single-line version info with format:
+// <version>-<git revision>-<go version>
```
Comment (gxthrj) [inline pkg/version/version.go:31]: Suggest to add information about the operating system to facilitate troubleshooting due to different build platforms.

Comment (tokers) [reply pkg/version/version.go:31]: You mean the runnning OS or the building OS?

Comment (gxthrj) [reply pkg/version/version.go:31]: building OS

Comment (tokers) [reply pkg/version/version.go:31]: OK, will add it.

Comment (tokers) [reply pkg/version/version.go:31]: All right, i changed my mind, the build os will confuse users when the binary was compiled crossly.

I will add a verbose version info, which contains this.

**Comment by tokers** [comment]
@gxthrj

**Comment by membphis** [comment]
@gxthrj ping

**Review by gxthrj — APPROVED** [review]
LGTM

**Pull request merged**